client/package.json

```diff
@@ -6,7 +6,16 @@
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
     "react-scripts": "2.1.8",
-    "styled-components": "^4.2.0"
+    "styled-components": "^4.2.0",
```

Comment:
Is this what you mentioned about eslint needing to be a dependency?

Reply:
no, `styled-components` is a package that we discuss to use instead of CSS directly.

those some of the dependances that makes the build errors `eslint-plugin-import`, `eslint-plugin-jsx-a11y` 

an example error at heroku build log:

```
 Creating an optimized production build...
       Failed to compile.
       
       ./src/index.js
       Error: Failed to load plugin import: Cannot find module 'eslint-plugin-import'
       Referenced from: 
           at Array.forEach (<anonymous>)
           at Array.reduceRight (<anonymous>)

```